mgmtsystem_action/models/mgmtsystem_action.py

```diff
@@ -184,14 +174,14 @@ def write(self, vals):
                     body=' %s ' % (_('Action cancelled on ') +
                                    fields.Datetime.now())
                 )
-        return super(MgmtSystemAction, self).write(vals)
+        return super(MgmtsystemAction, self).write(vals)
 
+    @api.multi
     def send_mail_for_action(self, action, force_send=True):
-        """Set a document state as draft and notified the reviewers."""
         template = self.env.ref(
             'mgmtsystem_action.email_template_new_action_reminder')
-        for action in self:
-            template.send_mail(action.id, force_send=force_send)
+        self.env['mail.template'].browse(template.id).send_mail(
```

Comment:
As template is a recordset there's no use of browsing mail.template.
You can simply use : template.send_mail()
Otherwise it makes an unnecessary select in DB.